cpp/include/cudf/text/tokenize.hpp

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/column/column.hpp>
+#include <cudf/scalar/scalar.hpp>
+#include <cudf/strings/strings_column_view.hpp>
+
+namespace cudf
+{
+namespace nvtext
+{
+
+/**
```

Comment:
Please avoid repeated documentation of the same functions in the top-level and detail namespaces by using @copydoc in the detail header. Just add the detail-level parameters to the docs, e.g.

```
/** @copydoc cudf::tokenize...
 *
 * @param mr ...
 * @param stream ...
 **/
```